e2e/benches/src/main.rs

```diff
@@ -29,23 +30,24 @@ async fn main() -> Result<()> {
     let logs = false;
     let kms = start_kms(logs).await?;
     let eth_node = start_eth(logs).await?;
-    let (main_key, secondary_key) = create_and_fund_kms_keys(&kms, &eth_node).await?;
+    let eth_signers = create_and_fund_kms_signers(&kms, &eth_node).await?;
+    let eigen_key = env!("EIGEN_KEY").to_string();
```

Comment:
nit: i think `env!` us a compile-time check; not a runtime check.

If this is not desired
```suggestion
    let eigen_key = std::env::var("EIGEN_KEY").expect("EIGEN_KEY environment variable must be set");
```